kubernetes-client/src/main/java/io/fabric8/kubernetes/client/internal/VersionUsageUtils.java

```diff
@@ -39,6 +39,10 @@ public static void log(String type, String version) {
       return;
     }
 
+    if (type.equals("customresourcedefinitions") && version.equals("v1beta1")) {
```

Comment:
In my opinion, we should not add checks like these here. Maybe print that alert in debug mode rather than warn?

Reply:
Yes, something I thought about. Can do it that way if you prefer.